source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,25 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.23.0:
+
+{+atlas-cli+} 1.23.0
+--------------------
+
+Released 23 May 2024
+
+- Adds support for Amazon Linux 2023. To learn more, see 
+  :ref:`install-atlas-cli`.
+
+- Updates :ref:`atlas-deployments-list` to list atlas clusters in case 
+  of local failures.
+
+- Stops publishing new versions on the ``server-4.4 Linux yum/apt`` 
+  repositories.
+
+- Adds support for ``srvShardOptimizedConnectionString`` to the 
```

Comment:
https://jira.mongodb.org/browse/CLOUDP-167925 -- this ticket was closed without the doc impact. Do you think we need a DOCSP? Because you are linking to a command that doesn't have this option, at the moment.

Reply:
I believe this is handled in the submodule.